StreamChatUITestsAppUITests/MockServer/MockServerAttributes.swift

```diff
@@ -67,11 +69,18 @@ enum JSONKey {
     static let member = "member"
     static let id = "id"
     static let cooldown = "cooldown"
+    static let watcherCount = "watcher_count"
+    static let imageAction = "image_action"
 
     enum Channel {
         static let addMembers = "add_members"
         static let removeMembers = "remove_members"
     }
+    
+    enum ImageActions {
```

Comment:
```suggestion
    enum AttachmentAction {
```

Reply:
Same concern as in the comment below. Let's discuss it.

Reply:
Updated 👍